### ISSUE TITLE
some effect on my Horizontal scroll bar

### ISSUE THREAD

**Issue by masontin** [jperasmus/docsify-copy-code#19]
Did your pages like this ?

```js
window.$docsify = {
…
      copyCode: {
        buttonText: '复制',
        errorText: '复制失败',
        successText: '已复制'
      },
…
    }
```

![1600678426290](https://user-images.githubusercontent.com/46433259/93748559-0ff92180-fc2b-11ea-9d45-65e03775ce7d.jpg)

**Comment by jperasmus** [comment]
Hmm, that is an interesting one. Do you perhaps have time to look into this and PR a fix? I currently have very limited time for this library.

**Comment by masontin** [comment]
> This problem can be reproduced on firefox and chrome, but not on safari. The reason is that the error/success message is longer than the label.
 
I had created a PR  #21  for this.  

And this is my test gif, there is no scroll bar anymore.

![Sep-21-2020 21-39-06](https://user-images.githubusercontent.com/46433259/93774279-a7726a80-fc53-11ea-9bac-16646258653e.gif)

**Comment by jperasmus** [comment]
Thanks for the PR. Your fix should be available on NPM as v2.1.1

**Comment by masontin** [comment]
Thanks, I just saw version 2.2.1 on NPM.
